docs/app/Components/ComponentDoc/ComponentExample.js

```diff
@@ -25,13 +25,15 @@ export default class ComponentExample extends Component {
 
   render() {
     let code = (
-      <Highlight className='language-javascript'>
-        {this.fileContents}
-      </Highlight>
+      <Column>
+        <Highlight className='language-javascript'>
+          {this.fileContents}
+        </Highlight>
+      </Column>
```

Comment:
Wrapping code in its own column separates it from the float issues.  Also adds some pleasant whitespace.